### ISSUE TITLE
Submit package to Melpa

### ISSUE THREAD

**Issue by Andre0991** [mmagnus/emacs-grammarly#5]
Making the package available on Melpa would make it easier to install on Emacs.

**Comment by mmagnus** [comment]
Yeah, that would be nice, at the moment I don't have time to learn how to do it ;-) @larstvei do you know how to do it and you are willing to do it?

**Comment by larstvei** [comment]
Sure!

I'll wait until we resolve #4 though. I'll let you know when it's done so that you can update the README, and maybe add a release so that it's available on melpa stable.

**Comment by mmagnus** [comment]
@larstvei  are we ready to go? Can you do it? :-) I have no idea how ;-)

**Comment by larstvei** [comment]
I'll give it a go as soon as I find the time 👍

**Comment by larstvei** [comment]
Hi! I was in the process of adding it but realized that the name `emacs-grammarly` is perhaps not ideal. For one, prefixing the name with `emacs` is not really a nice package name (very few other packages has emacs in the name). Another thing is that the name should mention osx, so that people won't be surprised when the package only works on os x.

A similar issue: melpa/melpa#2984

I'm thinking it is best to do a name change before adding it to melpa. My suggestion would be `send-to-osx-grammarly`. It requires that the `.el` file is renamed as well (to `send-to-osx-grammarly.el` if you choose to go for that name).

I also suggest you add a copyright and license to the `.el` file before submitting, you can find an example [here](https://web.archive.org/web/20111120220609/http://marmalade-repo.org/doc-files/package.5.html). Lastly, you should check out what [they'll look for](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md#expediting-recipe-reviews) before accepting.
